qa-test-apps/helm-app/values-from-with-empty-values/values.yaml

```diff
@@ -0,0 +1,2 @@
+key: key1
```

Comment:
Same comment as for `values-files-with-no-values`. Or is this file used to populate the `test-map` config map?

Reply:
Via `test-map`, it will get override. I am providing below `ConfigMap`:

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: test-map
  namespace: default
data:
  values.yaml: |
    key: mykey,
    value: values123
```

Reply:
Right, so that `test-map` config map is created from another source with different values, overriding values from this file. Understood, thanks for clarifying :+1: